### PR TITLE
fix assertion for windows

### PR DESCRIPTION
**Pull request by cos**
@radumereuta, please review.

### CODE REVIEW

**Comment by dwightguth** [comment]
Looks fine to me. You can merge if Radu approves and the tests pass.

**Comment by radumereuta** [comment]
But why isn't this failing on linux now?

**Comment by cos** [comment]
Because you never get `\r\n` on linux.

**Comment by radumereuta** [comment]
Right... Please add a comment explaining this, and you can merge.

File: k-distribution/src/test/java/org/kframework/kore/convertors/BaseTest.java [hunk 1]
```diff
@@ -71,7 +71,9 @@ private void testConversion(Function<File, DefintionWithContext> parse) throws I
             printWriter.print(COPYRIGHT_HEADER + sep + actualOutput + "\n");
             printWriter.close();
         } else {
-            String expectedOutput = FileUtils.readFileToString(kilExpectedDefinitionFile);
+            String expectedOutput = FileUtils.readFileToString(kilExpectedDefinitionFile).replaceAll("\r\n","\n"); 
+            // replacing line endings for those still stuck with Windows
```
Comment (grosu) [inline k-distribution/src/test/java/org/kframework/kore/convertors/BaseTest.java]: Please be polite in these public comments.  Nobody cares of your subjective opinion.

**Comment by grosu** [comment]
I don't know how to read this.  It says "Failed" above, so we cannot merge, but when I click on Details above it shows no failures.  It even shows a

```
Test Result (no failures)
```

**Comment by dwightguth** [comment]
If you read the console output, it failed because the checkstyle check failed on the file @cos modified.

@cos can you please confirm there is an actual bug in the whitespace of that file and it's not just somehow picking up the `\r\n`?

**Pull request merged**